libraries/AP_CheckFirmware/AP_CheckFirmware.cpp

```diff
@@ -14,25 +14,39 @@
 #include <string.h>
 #include "monocypher.h"
 
-struct PACKED secure_data {
-    uint8_t sig[8] = {0x4e, 0xcf, 0x4e, 0xa5, 0xa6, 0xb6, 0xf7, 0x29};
-    struct PACKED {
-        uint8_t key[32] = {};
-    } public_key[10];
-};
+const struct ap_secure_data public_keys __attribute__((section(".ecc_raw")));
```

Comment:
ecc_raw name should be changed to maybe public key or something, as we are not using ECC cryptography anymore.

Reply:
ok, will do

Reply:
fixed